From 02cce78752edb347074ccda24c7c4f6ea4b202e2 Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Wed, 18 May 2011 18:42:35 -0400
Subject: [PATCH 172/248] rcu-list-for-each-entry-continue-rcu

rcu list_for_each_entry_continue_rcu

Needed to continue a list_for_each_entry_rcu iteration.

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@polymtl.ca>
---
 include/linux/rculist.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/linux/rculist.h b/include/linux/rculist.h
index 3820223..5257264 100644
--- a/include/linux/rculist.h
+++ b/include/linux/rculist.h
@@ -292,6 +292,7 @@ static inline void list_splice_init_rcu(struct list_head *list,
 	     &pos->member != (head);	\
 	     pos = list_entry_rcu(pos->member.next, typeof(*pos), member))
 
+
 /**
  * hlist_del_rcu - deletes entry from hash list without re-initialization
  * @n: the element to delete from the hash list.
-- 
1.7.0.4

